composer.json

```
@@ -0,0 +1,48 @@
+{
+    "name": "ftw-soft/intercom-notification-channel",
+    "description": "Intercom Notifications Driver",
+    "homepage": "https://github.com/ftw-soft/intercom-notification-channel",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Andrey Telesh",
+            "email": "andrey.telesh.y@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.1.3",
+        "guzzlehttp/guzzle": "^6.3",
+        "intercom/intercom-php": "^3.2",
+        "illuminate/notifications": ">=5.5",
```

```suggestion
        "illuminate/notifications": "~5.5 || ~6.0",
```

laravel 7 will be a major version and may break. (and update constraint below)